apps/playground/src/stories/hooks/useHover.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+import { useHover } from '../../../../../packages/react-hooks/use-hover';
+// import { useHover } from "@nanlabs/react-hooks";
```

Comment:
is there any reason why you are not doing `import { useHover } from "@nanlabs/react-hooks";`?

Reply:
in case it does not work, you can update the file at `packages/react-hooks/index.ts` adding use-hover in there 😊

Reply:
[here](https://github.com/boredcity/nancy.js/blob/bdad49e7a82af9558e7a55231a83b0025e5cdebe/packages/react-hooks/index.ts#L15)!

Reply:
will do, thanks!